packages/js/components/package.json

```diff
@@ -107,6 +109,7 @@
 		"jest-cli": "^27.5.1",
 		"postcss-loader": "^3.0.0",
 		"require-turbo": "workspace:*",
+		"react": "^17.0.0",
```

Comment:
Why do we need to add "react" here? I think we have also included it in `peerDependencies`. ❓

Reply:
I had to look this up to understand why: https://blog.bitsrc.io/understanding-peer-dependencies-in-javascript-dbdb4ab5a7be

it looks like we use some React stuff directly so peerDeps isn't sufficient

Reply:
Got it! Thanks for looking into it.

Reply:
done!